nrtk_explorer/app/core.py

```diff
@@ -76,6 +79,10 @@ def __init__(self, server=None):
             "collapse_dataset", "collapse_embeddings", "collapse_filter", "collapse_transforms"
         )
 
+        self.state.horizontal_split = 17
+        self.state.vertical_split = 40
```

Comment:
No magic numbers, convert this to constants on the form of `HORIZONTAL_SPLIT_DEFAULT_VALUE`